src/layout/Footer.tsx

```diff
@@ -1,51 +1,31 @@
-import React, { FC } from 'react';
+import React from 'react';
 import styled from 'styled-components';
 import { useTranslation } from 'react-i18next';
-import logo from '../resources/images/unit_logo.png';
-import { Link } from 'react-router-dom';
-import { Link as MuiLink } from '@material-ui/core';
-import NormalText from '../components/NormalText';
+import { Button } from '@material-ui/core';
 import { UrlPathTemplate } from '../utils/urlPaths';
 
 const StyledFooter = styled.div`
   display: grid;
-  grid-template-areas: '. logo privacy';
-  grid-template-columns: 1fr 1fr 1fr;
-  min-height: 3rem;
+  grid-template-areas: '. privacy';
+  grid-template-columns: 4fr 1fr;
   align-items: center;
-  border-top: 2px solid ${({ theme }) => theme.palette.divider};
+  min-height: 4rem;
+  background: rgba(210, 210, 210, 0.25);
```

Comment:
Blir aldri helt klok på hvor vi bør legge sånne ting... Denne fargen skal bare brukes i footer, men syns likevel at det egentlig burde vært lagt i theme. 
Det aller mest naturlige ville vært `palette.background.footer` imo, men da må vi utvide `background` objektet til å ha `footer` også da, så syns det blir litt mye styr bare for en liten ting også... hva mener du @sarahse ?

Reply:
Jeg synes vi godt kan definere alle fargene vi bruker i theme. Uansett om vi bruker fargen en gang eller flere. Det er så mye enklere å endre på farger i ettertid da.

Reply:
Da flytter jeg det til `palette.background.footer`!